src/main/java/mate/academy/Main.java

```diff
@@ -44,12 +52,24 @@ public static void main(String[] args) {
         yesterdayMovieSession.setMovie(fastAndFurious);
         yesterdayMovieSession.setShowTime(LocalDateTime.now().minusDays(1L));
 
-        MovieSessionService movieSessionService = null;
+        MovieSessionService movieSessionService =
+                (MovieSessionService) injector.getInstance(MovieSessionService.class);
         movieSessionService.add(tomorrowMovieSession);
         movieSessionService.add(yesterdayMovieSession);
 
         System.out.println(movieSessionService.get(yesterdayMovieSession.getId()));
         System.out.println(movieSessionService.findAvailableSessions(
                         fastAndFurious.getId(), LocalDate.now()));
+
+        UserService userService = (UserService) injector.getInstance(UserService.class);
+        AuthenticationService authenticationService =
+                (AuthenticationService) injector.getInstance(AuthenticationService.class);
+        try {
+            userService.add(authenticationService.register("alice@gmail.com", "1234"));
+            userService.add(authenticationService.register("bob@gmail.com", "qwerty"));
+            System.out.println(authenticationService.login("alice@gmail.com", "1234"));
+        } catch (RegistrationException | AuthenticationException e) {
+            e.printStackTrace();
```

Comment:
This is a bad practice. If you are catching exception in place like main method - it's better to print out meaningful message